packages/typespec-ts/test/modularUnit/scenarios/apiOperations/apiOperations.md

```diff
@@ -585,25 +605,30 @@ op test(...ApiVersionParameter): string;
 ## Operations
 
 ```ts operations
-import { TestingContext as Client } from "./index.js";
+import { TestingContext as Client } from "../index.js";
 import {
   StreamableMethod,
   PathUncheckedResponse,
   createRestError,
-  operationOptionsToRequestParameters
+  operationOptionsToRequestParameters,
 } from "@azure-rest/core-client";
 
 export function _testSend(
   context: Client,
-  options: TestOptionalParams = { requestOptions: {} }
+  apiVersion: string,
```

Comment:
this should be expected as tcgc will not lift the api version to the client level by default if it's not using versioned decorators.